src/lib-components/MastheadSecondary.vue

```diff
@@ -0,0 +1,185 @@
+<script setup lang="ts">
+import { computed } from 'vue'
+import type { PropType } from 'vue'
+
+import _isEmpty from 'lodash/isEmpty'
+
+import ResponsiveImage from '@/lib-components/ResponsiveImage.vue'
+import RichText from '@/lib-components/RichText.vue'
+
+import type { ImageItemType } from '@/types/types'
+
+const props = defineProps({
+  hasMolecules: {
+    type: Boolean,
+    default: true,
+  },
+  title: {
+    type: String,
+    required: true,
+  },
+  text: {
+    type: String,
+    default: '',
+  },
+  theme: {
+    type: String,
+    default: '',
+  },
+  heroImage: {
+    type: Object as PropType<ImageItemType>,
+    default: () => ({}),
+  },
+})
+
+const hasHeroImage = computed(() => {
+  return !_isEmpty(props.heroImage)
+})
+
+const classes = computed(() => {
+  return [
+    'masthead-secondary',
+    {
+      'has-molecule': props.hasMolecules && !hasHeroImage.value,
+    },
+    // See TODO in MastheadSecondary.stories.js
+    // {
+    //   'has-slot': this.$slots.default,
+    // },
+    {
+      'theme-meap': props.theme.toLowerCase() === 'meap',
+    },
+  ]
+})
+</script>
+
+<template>
+  <div :class="classes">
+    <ResponsiveImage
+      v-if="hasHeroImage"
+      class="hero-image"
+      :image="heroImage"
+    />
+    <div class="container">
+      <div class="meta">
+        <h1 class="title" v-html="title" />
+        <RichText v-if="text" class="text" :rich-text-content="text" />
+      </div>
+    </div>
+  </div>
+</template>
+
+<style lang="scss" scoped>
+.masthead-secondary {
+  z-index: 0;
+  position: relative;
+  background: var(--gradient-03), var(--text-overlay);
+  .hero-image {
+      position: absolute;
+      width: 100%;
+      height: 100%;
+      filter: brightness(0.75);
+
+      :deep(.media) {
+          object-fit: cover;
+      }
+  }
+  .rich-text {
+      :deep(p) {
+          color: var(--color-white);
+      }
+  }
+
+  .container {
+      max-width: $container-l-cta + px;
+
+      position: relative;
+      z-index: 20;
+      margin: 0 auto;
+      padding-top: var(--space-xl);
+      padding-bottom: var(--space-4xl);
+  }
+  .meta {
+      color: var(--color-white);
+
+      display: flex;
+      align-items: flex-start;
+      justify-content: center;
+      flex-direction: column;
+      gap: var(--space-s);
+
+      .title {
+          @include step-5;
+          text-shadow: 0px 2px 8px rgba(113, 113, 113, 0.08);
+      }
+      .text {
+          @include step-0;
+          text-shadow: 0px 2px 8px rgba(113, 113, 113, 0.08);
+          margin: 0;
+      }
+  }
+  :deep(h3),
+  :deep(h4),
+  :deep(h5),
+  :deep(p),
+  :deep(li) {
+      color: var(--color-white);
+      margin: 0;
+  }
+
+  // Variants
+  &.has-molecule:before {
+      content: "";
+      position: absolute;
+      top: 0;
+      left: 0;
+      bottom: 0;
+      right: 0;
+      background-image: url("node_modules/ucla-library-design-tokens/assets/svgs/molecule-masthead.svg");
```

Comment:
update this to  background-image: url("ucla-library-design-tokens/assets/svgs/molecule-masthead.svg");

Reply:
Good catch. 👍🏽  Will watch out for this pattern going forward.